flows/triggers/manual.go

```diff
@@ -37,23 +37,29 @@ type ManualTrigger struct {
 
 // NewManual creates a new manual trigger
 func NewManual(env envs.Environment, flow *assets.FlowReference, contact *flows.Contact, params *types.XObject) flows.Trigger {
-	if params == nil {
```

Comment:
this logic needs to be in `Context()` so it applies to triggers loaded from JSON too